src/modules/app/containers/Switcher.js

```diff
@@ -31,8 +31,8 @@ import CityModel from '../../endpoint/models/CityModel'
 import LoadingSpinner from '../../common/components/LoadingSpinner'
 import Layout from '../../layout/components/Layout'
 import LocationLayout, { LocationLayoutRoutes } from '../../layout/containers/LocationLayout'
-import GeneralHeader from '../../layout/components/GeneralHeader'
-import GeneralFooter from '../../layout/components/GeneralFooter'
+import GeneralHeader from '../../layout/components/header/GeneralHeader'
```

Comment:
I think we should keep our flat structure. Its more scalable not to have a deeply nested sturcture. Also having up to 50 files in a folder should be no problem. Looks a bit much, almost all projects work with a structure like this.

Reply:
You can make a footer and a header module. Which depend on the layout module.